src/item.ts

```diff
@@ -16,7 +16,14 @@ interface ChildrenFnProps {
    *
    * Can be omitted if there is only one item in gallery
    */
-  ref: ItemRef
+  ref: (node) => void
```

Comment:
Not sure if we have typescript pipeline check. Looks like no type provided here and in useCallback